### PR TITLE
update samples to sbt 1.5.6 

### PR DESCRIPTION
**Pull request by octonato**
Curious to see if we can easily upgrade sbt version. 

I did a local tests with a seed template and worked well.

### CODE REVIEW

**Review by ihostage — APPROVED** [review]
LGTM

File: couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.5.5
```
Comment (ennru) [inline couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties]: Update to 1.5.6...

Comment (octonato) [reply couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties]: done

**Review by ennru — APPROVED** [review]
Only the latest!

File: couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.5.6
```
Comment (ennru) [inline couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties]: And today's bid is 1.5.7!

Comment (ihostage) [reply couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties]: We need a condition for exit from this loop 😂

**Comment by ennru** [comment]
Travis is sad because Jabba won't play with them.

**Comment by octonato** [comment]
I won't continue this, closing...

**Pull request closed without merging**